utils/build/docker/nodejs/express4/integrations/db/postgres.js

```diff
@@ -1,87 +1,85 @@
-"use strict";
+'use strict'
 
 const { join } = require('path')
-const { Client, Pool } = require('pg')
-const { readFileSync, statSync } = require('fs')
+const { Client } = require('pg')
+const { readFileSync } = require('fs')
 
-async function launchQuery(query) {
-    return new Promise(function (resolve, reject) {
-        const client = new Client()
-        client.connect().then(() => {
-            client.query(query).then((queryResult) => {
-                console.log("Postgres query result:" + queryResult);
-                resolve("Postgres query done!")
-            }).catch((err) => {
-                console.log(err)
-                resolve("Error on postgres query")
-            })
-        }).catch((err) => {
-            console.log(err)
-            reject("Error on postgres connection")
-        })
-    });
+async function launchQuery (query) {
+  return new Promise(function (resolve, reject) {
+    const client = new Client()
+    client.connect().then(() => {
+      client.query(query).then((queryResult) => {
+        console.log('Postgres query result:' + queryResult)
+        resolve('Postgres query done!')
+      }).catch((err) => {
+        console.log(err)
+        resolve('Error on postgres query')
+      })
+    }).catch((err) => {
+      console.log(err)
+      reject(err)
```

Comment:
Heads up: that's not a linting change

Reply:
it's the `prefer-promise-reject-errors` rule.
Do you mean it could change the app behavior?

Reply:
Yes, idk if it's important or not, but just a heads up for APM to check if they actually want this change or not